base/replutil.jl

```diff
@@ -116,7 +116,7 @@ function showerror(io::IO, ex::MethodError)
     f = ex.f
     meth = methods_including_ambiguous(f, arg_types)
     if length(meth) > 1
-        return showerror_ambiguous(io, meth, f, arg_types)
+        return showerror_ambiguous(io, map(x->x[3], meth), f, arg_types)
```

Comment:
@JeffBezanson, or would you rather change this in `methods_including_ambiguous`?